packages/tools/lib/register.js

```diff
@@ -34,6 +34,7 @@ register(module.exports = {
     npm: {
       '': '@nikitajs/tools/lib/npm',
       uninstall: '@nikitajs/tools/lib/npm/uninstall'
-    }
+    },
+    apm: '@nikitajs/tools/lib/apm'
```

Comment:
why is this the only js generated file, there must be apm.js as well

Reply:
Maybe I don't understand your comment. 
`@nikitajs/tools/lib/apm' is in `lib/register.js` and `@nikitajs/tools/src/apm' is in `src/register.coffee.md``

Reply:
It is related to running `yarn run build`. If I remember well, I meant is that I only say 'lib/register.js` being transpiled and not `lib/apm.js`. However, I just checked on you branch and `lib/apm.js` seems to be there.